docs/index.md

```diff
@@ -1,25 +1,49 @@
-# Introduction
+# Introduction to `cfgov-refresh`
```

Comment:
I see that this is styled this way below, but I don't think we usually backtick repo/project names.